subprojects/docs/src/docs/userguide/artifact_management.adoc

```diff
@@ -13,95 +13,99 @@
 // limitations under the License.
 
 [[artifact_management]]
-== Legacy publishing
-
-[NOTE]
-====
-
-This chapter describes the _original_ publishing mechanism available in Gradle 1.0, which has since been superseded by <<publishing_overview,an alternative model>>. The approach detailed in this chapter — based on api:org.gradle.api.tasks.Upload[] tasks — should not be used in new builds. We cover it in order to help users work with and update existing builds that use it.
-
-====
+= Legacy publishing
 
+NOTE: This chapter describes the _original_ publishing mechanism available in Gradle 1.0, which has since been superseded by <<publishing_overview.adoc#publishing_overview,an alternative model>>. The approach detailed in this chapter — based on link:{groovyDslPath}/org.gradle.api.tasks.Upload.html[Upload] tasks — should not be used in new builds. We cover it in order to help users work with and update existing builds that use it.
 
 [[sec:introduction]]
-=== Introduction
+== Introduction
 
 This chapter is about how you declare the outgoing artifacts of your project, and how to work with them (e.g. upload them). We define the artifacts of the projects as the files the project provides to the outside world. This might be a library or a ZIP distribution or any other file. A project can publish as many artifacts as it wants.
 
 [[sec:artifacts_and_configurations]]
-=== Artifacts and configurations
+== Artifacts and configurations
 
 Like dependencies, artifacts are grouped by configurations. In fact, a configuration can contain both artifacts and dependencies at the same time.
 
 For each configuration in your project, Gradle provides the tasks `upload__ConfigurationName__` and `build__ConfigurationName__`.footnote:[To be exact, the Base plugin provides those tasks. This plugin is automatically applied if you use the Java plugin.] Execution of these tasks will build or upload the artifacts belonging to the respective configuration.
```

Comment:
Due to changes in the formatting, this footnote now appears in an odd place in the paragraph:

![image](https://user-images.githubusercontent.com/5387972/42967609-cb45e1c4-8b6e-11e8-999f-8c7af52bb186.png)

Can we just incorporate the foot note into the text? Just say...
> For each configuration in your project, Gradle provides the tasks uploadConfigurationName and buildConfigurationName when the base plugin is applied[link to base plugin].

Reply:
Agreed and changed roughly as you suggest.